examples/datadog-express-apm/host-agent/utils/response.js

```diff
@@ -30,7 +30,7 @@ class ResponseHandler {
     agent.init().dogstatsd.increment(`Commercetools_Client_Response`, 1, {
       env: 'dev',
       status_code: statusCode,
-      http_method: req.method,
+
```

Comment:
```suggestion
http_method: res.req.method,
```

Reply:
Oh I didn't see this anyway I did the same liek your suggestion ;)